### ISSUE TITLE
Data used to train pretrained model

### ISSUE THREAD

**Issue by xxxpsyduck** [courao/ocr.pytorch#11]
Can you describe more about the data used to train your pretrained model? About the language, number of samples, label format...

**Comment by courao** [comment]
You mean CRNN or CTPN?
I'll update description for training the CRNN part later.

**Comment by xxxpsyduck** [comment]
Oh I don't expect that we have to train 2 part separately
